### PR TITLE
[WIP] Release prep

### PR DESCRIPTION
**Pull request by fgrehm**
This PR introduces a RubyGem package for Vital and drops middleman for our main assets build process (it is only used to build the website / docs for the framework), Sass variables were also updated so that they are set as `!default`s

## TODO

- [x] Separate website from assets to be distributed
- [x] Support for `font-url` (get from twbs / fontawesome)
- [x] Gem package with support for rails
- [x] Task for rebuilding font icons
- [x] Fine tune what gets included on the gem package (no website nor raw icons)
- [x] Rake task to prepare package for a "tarball release"
- [x] ~~Social share icons?~~ not needed
- [x] Find a way to reference the components page from rails apps
- [ ] Sample app to act as a live demo of rails integration
- [ ] Automatically upload tarballs to github
- [ ] Update docs
- [ ] Revert font naming from `vital-icons` to `icons`
- [ ] Update file size in documentations site
- [ ] Update documentation to reflect `/website/` to run documentation server

### CODE REVIEW

File: assets/stylesheets/vital/_icons.sass [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+.icon-close-outline:before
+  content: "\f104"
+
+.icon-menu:before
+  content: "\f10a"
+
+.icon-vital:before
+  content: "\f101"
+
+$font-vital-icons-icon-arrow-updown: ""
```
Comment (bodytaing) [inline assets/stylesheets/vital/_icons.sass]: Should use the character set "/foo" etc.

**Comment by bodytaing** [comment]
👍  Lots of great work here @fgrehm. I'll jump in and make some small changes. We can probably forego the inclusion social icons - want to leave this up to the implementer that will use Vital.

File: assets/stylesheets/vital/_icons.sass (hunk 1 above)
Comment (fgrehm) [inline assets/stylesheets/vital/_icons.sass]: Oh, this _might_ come from the `sed` we run to tweak font urls 💭

Comment (fgrehm) [inline assets/stylesheets/vital/_icons.sass]: This is actually caused by sass / sass-convert and I believe is related to https://github.com/sass/sass/issues/1395 😞 I found a way around it by digging through those comments and related issues but unfortunately I closed the tab that has the fix after getting it in place for this project so I can't give you a link with more info on it 🐼

Fix for this is the not so pretty 080f50a

**Comment by brupm** [comment]
@jeyb

**Comment by fgrehm** [comment]
Closing this in favor of all the other smaller PRs I sent out along the way:

- https://github.com/doximity/vital/pull/24
- https://github.com/doximity/vital/pull/25
- https://github.com/doximity/vital/pull/28

**Pull request closed without merging**